test/e2e/autoscale_test.go

```diff
@@ -477,3 +479,53 @@ func TestTargetBurstCapacityMinusOne(t *testing.T) {
 		t.Fatalf("Initial state never achieved: %v", err)
 	}
 }
+
+func TestFastScaleToZero(t *testing.T) {
+	t.Parallel()
+	cancel := logstream.Start(t)
+	defer cancel()
+
+	ctx := setup(t, autoscaling.KPA, autoscaling.Concurrency,
```

Comment:
It is not clear from the method name what is setup or in what state things are when it returns.  Are they ready?  Are they freshly created?  I only know that a Service is created from the params, which is not ideal :(

Reply:
I've described the `setup` separately.